src/Files.App/ViewModels/HomeViewModel.cs

```diff
@@ -1,80 +0,0 @@
-using CommunityToolkit.Mvvm.ComponentModel;
```

Comment:
Why did you renamed Home to Widgets? To be consistent, Both Page and ViewModel names should have the same.